c2corg_api/views/document.py

```diff
@@ -30,3 +31,64 @@ def _create_new_version(self, document):
         DBSession.add(meta_data)
         DBSession.add_all(versions)
         DBSession.flush()
+
+    def _update_version(self, document, comment, update_type, changed_langs):
+        assert update_type != UpdateType.NONE
+
+        meta_data = HistoryMetaData(comment=comment)
+        archive = self._get_document_archive(document, update_type)
+
+        cultures = \
+            self._get_cultures_to_update(document, update_type, changed_langs)
+        locale_versions = []
+        for culture in cultures:
+            locale = document.get_locale(culture)
+            locale_archive = self._get_locale_archive(locale, changed_langs)
+
+            version = DocumentVersion(
+                document_id=document.document_id,
+                culture=locale.culture,
+                version=999,
```

Comment:
This is actually still a *TODO*. I am not sure how to increment the version in a safer manner. Maybe just drop the version column and calculate a number (like in this [answer](http://dba.stackexchange.com/a/47794))? Or use a version hash?